deploy/user-role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    kubernetes.io/bootstrapping: rbac-defaults
```

Comment:
You shouldn't apply this annotation yourself.  It's for bootstrap roles and this is an add-on